src/plugin/cesium/sync/featureconverter.js

```diff
@@ -1035,13 +1037,20 @@ plugin.cesium.sync.FeatureConverter.prototype.olPolygonGeometryToCesiumPolyline
 
 
 /**
- * @param {boolean} enabled
+ * @param {os.webgl.AltitudeMode} altitudeMode
  */
-plugin.cesium.sync.FeatureConverter.prototype.setAltitudeEnabled = function(enabled) {
-  if (enabled) {
-    this.heightReference_ = Cesium.HeightReference.NONE;
-  } else {
-    this.heightReference_ = Cesium.HeightReference.CLAMP_TO_GROUND;
+plugin.cesium.sync.FeatureConverter.prototype.setAltitudeMode = function(altitudeMode) {
+  switch (altitudeMode) {
+    case os.webgl.AltitudeMode.ABSOLUTE:
+      this.heightReference_ = Cesium.HeightReference.NONE;
+      break;
+    case os.webgl.AltitudeMode.RELATIVE_TO_GROUND:
+      this.heightReference_ = Cesium.HeightReference.RELATIVE_TO_GROUND;
+      break;
+    case os.webgl.AltitudeMode.CLAMP_TO_GROUND:
+    default:
+      this.heightReference_ = Cesium.HeightReference.CLAMP_TO_GROUND;
```

Comment:
The layer default should be `ABSOLUTE`, since `CLAMP_TO_GROUND` will incorrectly display features that are actually above/below the ground.

Later we can address having our parsing stack set individual features to `CLAMP_TO_GROUND` if they don't have an altitude.